rebar.config

```diff
@@ -17,7 +17,8 @@
 
 {deps, [{katana_code, "1.1.1"}]}.
 
-{project_plugins, [rebar3_lint, rebar3_hank, rebar3_hex]}.
+{project_plugins,
+ [{rebar3_hex, "~> 6.11.0"}, {rebar3_lint, "~> 0.4.0"}, {rebar3_hank, "~> 0.3.0"}]}.
```

Comment:
@elbrujohalcon, just a small heads-up. Mind you that `rebar3_lint`'s bump to a future `0.4._` might break interface, as per semver rules 😄

Reply:
We're aware. Thanks.

Reply:
I'd never do it to you, though 😄 and I hope we reach `1.0.0` soonish (in `rebar3_lint`) since the plugin's basically complete, and now updates are a question of up'ing `elvis_core`.

Reply:
And if the plugin did stuff like `~> 1`, as long as we didn't break `elvis_core` (meaning respecting semver), it would most likely only get updated when `elvis_core` 2.x came along. (which is not in the current plans, IIRC).